trunk/src/modules/paraview.F90

```diff
@@ -30,6 +33,11 @@ module paraview
    integer, allocatable :: GEOM_OBJ(:)
    real(8), allocatable :: GEOM_PTS(:,:)
    real(8), allocatable :: ATTR_VAL(:,:)
+
+! Variables for VTU Format
+   integer, allocatable    :: IPARATTR_VTU(:)
+   integer(8),    allocatable :: ELEM_TYPES(:)
```

Comment:
Style:
```suggestion
!..Variables for VTU Format
   integer   , allocatable :: IPARATTR_VTU(:)
   integer(8), allocatable :: ELEM_TYPES(:)
```

Reply:
Are long integers needed here?

Reply:
Changes implemented: no need of long integers